package.json

```diff
@@ -7,6 +7,12 @@
   "repository": "git@github.com:CanopyTax/bandicoot.git",
   "author": "Joel Denning <joeldenning@gmail.com>",
   "license": "MIT",
+  "keywords": [
```

Comment:
After we add keywords, [this search](https://www.npmjs.com/search?q=react%20rich%20text) should hopefully show bandicoot